app/sessions.go

```diff
@@ -0,0 +1,53 @@
+package app
+
+import (
+	"database/sql"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+type Session struct {
+	UserId    int
+	SessionId int
+	ExpiresAt time.Time
+}
+
+type SessionService struct {
+	db *sql.DB
+}
+
+func NewSessionService(db *sql.DB) *SessionService {
+	return &SessionService{
+		db: db,
+	}
+}
+
+func (s *SessionService) SaveSession(userID int, expiresAt time.Time) (int, error) {
+	var sessionID int
+	err := s.db.QueryRow("INSERT INTO sessions (user_id, expires_at) VALUES ($1, $2) RETURNING session_id", userID, expiresAt).Scan(&sessionID)
+	if err != nil {
+		return 0, err
+	}
+	return sessionID, nil
+}
+
+func (s *SessionService) SetSessionCookie(w http.ResponseWriter, sessionID int) {
+	http.SetCookie(w, &http.Cookie{
+		Name:    "session_id",
+		Value:   strconv.Itoa(sessionID),
+		Expires: time.Now().Add(24 * time.Hour),
```

Comment:
Instead of using a magical number `24`, you can create a constant `const SessionDurationInHours = 24` in the `app` package. You can then use this constant when you move this implementation to the `server` pacakge (`app.SesssionDurationInHours`).